### PR TITLE
add geo tables

### PR DESCRIPTION
**Pull request by kmagida110**
## Background
This PR converts our geography tables from binary shape files to text that is readable as json.

### CODE REVIEW

File: src/analytics_tables/geography_tables.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
+create table uploaded_data.ct_town_geo
```
Comment (imcurtis) [inline src/analytics_tables/geography_tables.sql:1]: Does the data come in with those column names or are you assigning them here along with dtype?

File: src/build_db.py [hunk 2]
```diff
@@ -133,3 +127,6 @@ def init_database(init_postgis: bool=False):
     print("Getting historical C4K data")
     get_historical_c4k(final_filename=f"{DB_DATA_FOLDER}/all_c4k_data.csv")
 
+    # Create shapefile CSVs
+    create_shapefile_csvs()
```
Comment (imcurtis) [inline src/build_db.py:131]: Does the create_shapefile_csvs function need an argument when it's run within init_database?

Comment (kmagida110) [reply src/build_db.py:131]: I don't think it is run inside init_database anywhere

**Review by imcurtis — COMMENTED** [review]
Looks good but waiting to confirm

File: src/data_integration/census_data/setup_geo_json.py [hunk 3]
```diff
@@ -92,4 +53,8 @@ def load_level_table(geo_level, table_name, columns, engine, file_type=CARTO):
         geo_leg_key = FINAL_HOUSE_ID if geo_level == HOUSE else FINAL_SENATE_ID
         level_geo_df = level_geo_df.merge(legis_df, how='left', left_on=geo_leg_key, right_on=CURRENT_DISTRICT)
 
-    write_to_sql(table_name=table_name, geo_df=level_geo_df, engine=engine, columns=columns)
+    # Convert geometries to JSON strings that are readable by Mapbox (in Superset)
+    level_geo_df[GEOMETRY_COL] = level_geo_df[GEOMETRY_COL].apply(convert_to_poly)
+    json_geometries = [json.dumps({'geometry': town['geometry']}) for town in json.loads(level_geo_df[GEOMETRY_COL].to_json())['features']]
```
[the remark this answers is not in the document]
Comment (kmagida110) [reply src/data_integration/census_data/setup_geo_json.py:58]: Close, the level_geo_df['geometry'] column is a json string, with a list in ['features'] for each town. This makes the string into a dictionary and pulls each town's geometry out of the dictionary and saves it in the right format (overwriting the existing geometry column) to be used as geojson.

File: README.md [hunk 4]
```diff
@@ -18,22 +18,16 @@ This instance of Superset is configured with the file in `config/superset_config
...
-When the database is initially built it needs to be set up with the following steps:
-
-1. Upgrade and initialize database from command line of Superset instance
-   - `docker exec -it superset superset db upgrade && docker exec -it superset superset init`
-1. Create admin user from command line of Superset instance
-   - `docker exec -it superset superset fab create-admin \
-                 --username USERNAME \
-                --firstname FIRSTNAME \
-                --email EMAIL \
-                --password ********`
```
Comment (bamader) [inline README.md:30]: Why are we deleting these instructions? Are we just scrapping all the docker based functionality?

File: src/build_db.py (hunk 2 above)
Comment (bamader) [reply src/build_db.py:131]: Yeah I'm not seeing that, Alex. It also doesn't have an argument signature so this should be fine.

File: src/data_integration/census_data/setup_geo_json.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (bamader) [reply src/data_integration/census_data/setup_geo_json.py:58]: Oh the joys of parsing json files

**Review by bamader — APPROVED** [review]

**Pull request merged**